charts/rosa-capi/values.yaml

```diff
@@ -1,5 +1,11 @@
 # Default values for rosa-capi.
 
+# Set the value to true if you need ArgoCd to be installed on the new cluster
+enable_argocd: false
```

Comment:
Let's standardize on camelCase as that's already in-use - i.e.: `enableArgoCD`

Reply:
fixed